examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -28,6 +28,7 @@ def main(cfg: DictConfig) -> None:
     trainer = pl.Trainer(**cfg.trainer)
     exp_manager(trainer, cfg.get("exp_manager", None))
     question_answering_model = QAModel(cfg.model, trainer=trainer)
+    question_answering_model.restore_from(cfg.model.checkpoint)
```

Comment:
restore_from is a class method. It will return model instance. Not sure 
```question_answering_model.restore_from(cfg.model.checkpoint)```
actually affects ```question_answering_model instance```

Reply:
i made it an instance method and overwrote it